docs/get-support.md

```diff
@@ -4,22 +4,33 @@ title: Get Support
 
 # Get Support
 
-The best way to request support from the Event Bus Team is to reach out to us via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the Office of CTO @VA workspace.
+## General support requests
 
-Please avoid direct messaging an Event Bus team member in Slack. Chatting openly in the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) channel allows others to learn from the questions and answers discussed.
+The best way to request support from the Event Bus Team is to reach out to us via the [#event-bus-support][slack-#event-bus-support] Slack channel in the Office of CTO @VA workspace.
 
-## Slack channel support hours 
+Please avoid direct messaging an Event Bus team member in Slack. Chatting openly in the [#event-bus-support][slack-#event-bus-support] channel allows others to learn from the questions and answers discussed.
 
-Event Bus Slack channel support hours are from 9am - 5pm ET, Monday through Friday, with the exception of federal holidays.
+Event Bus Slack channel support hours are from 12pm - 5pm ET, Monday through Friday, with the exception of federal holidays. Often, an engineer will respond by 9am ET if they live in the Eastern or Central timezones. You can expect a response within 1 hour during the support hours.
 
-Response times are based on the nature of the posting:
+## Report an incident
 
-* Incidents (outage or error): ASAP. Please clearly indicate this is an “INCIDENT” in the post.
-* General inquiries: 2-3 hours.
+To report a security or privacy incident, or an urgent critical operational incident email event-bus@veterans-affairs.pagerduty.com. This will generate a page to the on-call engineer's mobile device 24/7. You can expect a response within 30 minutes at all hours.
```

Comment:
Would it be "all hours." or "any hours"? English is hard

Reply:
If it's 24/7, I think you can remove "at all hours."